City-of-Helsinki/drupal-hdbt
# UHF-3503: News item styling

# [UHF-3503](https://helsinkisolutionoffice.atlassian.net/browse/UHF-3503)

## Please notice!

This will be merged to the UHF-3268_news_item branch instead of 1.x

## What was done

* Generic fixes
  * Set default font color for page
  * Moved heading font-sizes to mixins for re-use purposes
  * Allow setting the display property of pseudo-icon mixin, so that it can be `block ´instead of `inline-block` where needed.
  * Renamed social media classes to be more BEM-like
  * Fixed figure margins and other image styling
* Styled news page with the aim of re-using this code in other pages at a later time
* Added translations

## How to test
* Get UHF-3268_news_item branch [from its PR](https://github.com/City-of-Helsinki/drupal-hdbt/pull/210)
* Check that the news styling works as instructed in [UHF-3503](https://helsinkisolutionoffice.atlassian.net/browse/UHF-3503)
* Check that other code improvements seem sane

Related: https://github.com/City-of-Helsinki/drupal-hdbt/pull/210
